internal/testrunner/compiler_runner.go

```diff
@@ -341,6 +343,33 @@ func (c *compilerTest) verifyDiagnostics(t *testing.T, suiteName string, isSubmo
 	})
 }
 
+func (c *compilerTest) verifyJavaScriptOutput(t *testing.T, suiteName string, isSubmodule bool) {
```

Comment:
This same thing will do dts files too, right? Should we just say it's "output"?

Reply:
This is what it was called in Strada. I don't see any sense in changing it while we're still porting/migrating as it makes it easier to find.